src/components/cast/_cast.scss

```diff
@@ -8,12 +8,22 @@
     background-color: rgba(255, 255, 255, 0);
     border: none;
     cursor: pointer;
-    --connected-color: rgba(1, 172, 205, 0.8);
+    --connected-color: #{$grayscale4};
     --disconnected-color: #{$grayscale4};
+
+    &:hover {
+      --connected-color: #{$white};
+      --disconnected-color: #{$white};
+    }
   }
 
-  .cast-button:hover {
-    --disconnected-color: #{$white};
-    --connected-color: #{$brand-color};
+  .cast-button.cast-button-active {
+    --connected-color: rgba(1, 172, 205, 0.8);
+    --disconnected-color: #{$grayscale4};
```

Comment:
why is needed? same for hover

Reply:
why is what needed

Reply:
`--disconnected-color: #{$grayscale4};` already set

Reply:
oh, you're right